src/main/java/net/minecraftforge/event/entity/living/EnderManAngerEvent.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.world.entity.monster.EnderMan;
+import net.minecraft.world.entity.player.Player;
+import net.minecraftforge.eventbus.api.Cancelable;
+
+/**
+ * This event is fired on the forge bus before an Enderman detects that a player is looking at them.
+ * It will not be fired if the detection is already prevented by {@link IForgeItem#isEnderMask}
+ * <p>
+ * This event is {@link Cancelable}.
+ * If this event is canceled, the Enderman will not target the player.
+ * <p>
+ * This event does not have a {@link Result}.
+ */
+@Cancelable
+public class EnderManAngerEvent extends LivingEvent {
```

Comment:
Formatting: braces on newlines.
```suggestion
public class EnderManAngerEvent extends LivingEvent
{
```